lib/kvdb/ikvdb.c

```diff
@@ -76,9 +76,12 @@
 
 /* clang-format off */
 
-static_assert((sizeof(uintptr_t) == sizeof(u64)),
+static_assert((sizeof(uintptr_t) == sizeof(uint64_t)),
```

Comment:
It would be useful to accompany this assertion with what parts of HSE assume this.